src/org/thoughtcrime/securesms/util/LongClickCopySpan.java

```diff
@@ -52,22 +49,20 @@ private void openChat(Activity activity, DcContact contact) {
   public void onClick(View widget) {
     String url = getURL();
     if (url.startsWith(PREFIX_CMD)) {
-	try {
-	    String cmd = url.substring(PREFIX_CMD.length());
-	    Activity activity = (Activity) widget.getContext();
-	    DcContext dcContext = DcHelper.getContext(activity);
-	    dcContext.sendTextMsg(this.chatId, cmd);
-	}
-	catch(Exception e) {
-	    e.printStackTrace();
-	}
+      try {
+        String cmd = url.substring(PREFIX_CMD.length());
+        ConversationActivity activity = (ConversationActivity) widget.getContext();
+        activity.setDraftText(cmd + " ");
```

Comment:
I am only setting the text as a draft to avoid accidental messages and to enable arguments (like /xkcd **30**).

Reply:
From @adbenitez 

> about setting as draft instead of sending directly, I think telegram sends them directly, for arguments there is dynamic commands like 
/xkcd_30
and anyway in long click the command is copied to clipboard

Reply:
I still think copying them is ok, sending the message on behalf of the user might be unexpected.